docs/creating_review_app.md

```diff
@@ -1,35 +1,14 @@
-## Creating a Review App
+## Creating a Force Review App
 
-If wanting to create a deploy for a WIP feature or for QA, [Hokusai](), supports [Review Apps](https://github.com/artsy/hokusai/blob/master/docs/Review_Apps.md).
+If you want to create a deploy for a WIP feature or for QA, [Hokusai](), supports [Review Apps](https://github.com/artsy/hokusai/blob/master/docs/Review_Apps.md).
```

Comment:
Commas not needed in `QA, [Hokusai](), supports`

Reply:
fixed. Thanks!